src/model/position.js

```diff
@@ -502,6 +502,18 @@ export default class Position {
 		}
 	}
 
+	/**
+	 * Checks whether given object is of `position` type.
+	 *
+	 * Read more at {@link module:engine/model/node~Node#is `Node#is()`}.
```

Comment:
Sending people to `Node#is()` from `Position` (which isn't a node) seems confusing.

Reply:
But see my other comment below about mentioning that this is a convention.